gammapy/scripts/config/schema.yaml

```diff
@@ -186,6 +186,57 @@ properties:
             dataset-type:
                 enum: [not assigned, SpectrumDatasetOnOff, MapDataset]
             containment_correction: {type: boolean}
+            offset-max: {type: string}
+            geom:
+                type: object
+                additionalProperties: false
+                properties:
+                    binsz: {type: number}
+                    coordsys:
+                        type: string
+                        enum: [CEL, GAL]
+                    proj: {type: string}
+                    skydir:
+                        type: array
+                        items: {type: number}
+                        minItems: 2
+                        maxItems: 2
+                    width:
+                        type: array
+                        items: {type: number}
+                        minItems: 1
+                        maxItems: 2
+                    axes:
+                        type: array
+                        additionalProperties: false
+                        items:
+                            "$ref": "#/definitions/map_axis"
+                required: [axes]
```

Comment:
Do we need to add more required properties other than `axes` or remove the requirement on this one? Note that as it is defined here `axes` will only be required if `geom` is not empty. We also define below conditional requirements for `geom` when `dataset-type` is a `MapDataset`